regen/feature.pl

```diff
@@ -761,6 +762,16 @@ =head2 The 'isa' feature
 
 This feature is available from Perl 5.32 onwards.
 
+=head2 The 'indirect' feature
+
+This feature allows the use of indirect method call syntax, e.g.
+C<new Foo 1, 2;>. It is enabled by default, but can be turned off to
+disallow indirect method calls.
+
+This feature is available under this name from Perl 5.32 onwards. In
+previous versions, it was simply on all the time.  To disable indirect
```

Comment:
It doesn't really disable it, just warns or dies when encountered.

Reply:
How about "disallow or warn on"?

Reply:
:+1: